frontend/src/app/modules/autocompleter/op-autocompleter/op-autocompleter.component.sass

```diff
@@ -0,0 +1,39 @@
+.op-autocompleter--option-wrapper
```

Comment:
According to our BEM implementation, please start every BEM compatible component sass with the block element, e.g., `.op-autocompleter`.

Then you can use the sass helpers to generate the elements:

```sass
.op-autocompleter

  // This will become .op-autocompleter--wp-content
  &--wp-content

```